src/docs/asciidoc/tcp-client.adoc

```diff
@@ -439,3 +476,79 @@ public class Application {
 }
 ----
 ====
+
+== Metrics
+TCP client supports built-in integration with https://micrometer.io/[`Micrometer`].
+It exposes all metrics with prefix `reactor.netty.tcp.client`.
+
+The following table provides information for the TCP client metrics:
+
+[width="100%",options="header"]
+|=======
+| metric name | type | description
+| reactor.netty.tcp.client.data.received | DistributionSummary | Amount of the data that is received, in bytes
+| reactor.netty.tcp.client.data.sent | DistributionSummary | Amount of the data that is sent, in bytes
+| reactor.netty.tcp.client.errors | Counter | Number of the errors that are occurred
+| reactor.netty.tcp.client.tls.handshake.time | Timer | Time that is spent for TLS handshake
+| reactor.netty.tcp.client.connect.time | Timer | Time that is spent for connecting to the remote address
+| reactor.netty.tcp.client.address.resolver | Timer | Time that is spent for resolving the address
+|=======
+
+These additional metrics also will be available:
+
+include::conn-provider-metrics.adoc[]
+
+include::alloc-metrics.adoc[]
+
+The following example enables that integration:
+
+====
+[source,java]
+----
+import reactor.netty.Connection;
+import reactor.netty.tcp.TcpClient;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Connection connection =
+                TcpClient.create()
+                         .host("example.com")
+                         .port(80)
+                         .metrics(true) <1>
+                         .connectNow();
+
+        connection.onDispose()
+                  .block();
+    }
+}
+----
+<1> Enables the built-in integration with Micrometer
+====
+
+When TCP client metrics are needed, but with an integration with a system different than `Micrometer` or you want
```

Comment:
Change it to the following: When HTTP client metrics are needed for an integration with a system other than Micrometer or you want
to provide your own integration with Micrometer, you can provide your own metrics recorder, as follows:

(Same comment as before)